docs/source/Getting-Started/Tutorials/Security-Monitor-Platform-Build.rst

```diff
@@ -0,0 +1,119 @@
+Tutorial 1: Build Keystone Security Monitor
```

Comment:
+ This should probably just be a build doc for the SM, rather than a
   tutorial
 + "Use it to boot the machine." ?
 + Use the default platform as example, rather than the waymasking code
   - Waymasking and cache partitioning isn't a core part of this
     tutorial process, it should probably go. If anywhere, it should
     go in its own docs, probably not a tutorial.
   - You can show how to specify the fu540, and mention what it does,
     but no need for detail

Reply:
Can you make changes for this doc? You implemented this part, so you'd explain it more clearly.

Reply:
Sure, I'll move those around